locales/lt-LT/messages.po

```diff
@@ -0,0 +1,1833 @@
+msgstr "Maksimalus kiekis variantų, kurį kandidatui leidžiama pasirinkti."
+
+msgid "The maximum number of choices that the candidate is required to select to form a valid response."
+msgstr "Maksimalus kiekis pasirinkimų, kuriuos kandidatas privalo pasirinkti, norint suformuoti taisyklingą atsakymą."
+
+msgid "The maximum number of choices this choice may be associated with."
+msgstr "Maksimalus kiekis variantų, kurių šiam variantui galima asocijuoti."
+
+msgid "The maximum number of choices this choice may be associated with. If matchMax is 0 there is no restriction."
+msgstr "Maksimalus kiekis pasirinkimų, kuriuos galima susieti. Jeigu matchMax yra 0, nėra jokio apribojimo."
+
+msgid "The maxPlays attribute indicates that the media object can be played at most maxPlays times - it must not be possible for the candidate to play the media object more than maxPlay times. A value of 0 (the default) indicates that there is no limit."
+msgstr ""
+
+msgid "The MIME-type attribute describes which kind of file may be uploaded."
+msgstr "MIME-tipo atributas nusako, kokia rūšis failo gali būti įkelta."
+
+msgid "The minimum number of answers is "
+msgstr "Minimalus kiekis atsakymų yra"
+
+msgid "The minimum number of associations that the candidate is required to make to form a valid response."
+msgstr "Minimalus kiekis ryšių, kurių kandidatui reikia sudaryti norint pateikti teisinga atsakymą."
+
+msgid "The minimum number of choices that the candidate is required to select to form a valid response."
+msgstr "Minimalus kiekis pasirinkimų, kuriuos kandidatas privalo pasirinkti, kad būtų suformuotas teisingas atsakymas."
+
+msgid "The minimum number of choices this choice must be associated with to form a valid response."
+msgstr "Minimalus kiekis pasirinkimų, kurie privalo būti susieti, kad suformuoti teisingą atsakymą."
+
+msgid "The number base to use when converting integer variables to strings with the i conversion type code."
+msgstr ""
+
+msgid "The number of units corresponding to a step on the slider"
+msgstr "Vieneto kiekis atitinkantis žingsnį slankikliui"
+
+msgid "The path argument for extractor must be a non-empty array."
+msgstr "Kelio argumentas ekstraktoriui privalo būti ne tuščias masyvas."
+
+msgid "The principle identifier of the item. This identifier must have a corresponding entry in the item's metadata."
+msgstr "Pagrindinis elemento identifikatorius. Šis identifikatorius privalo turėti papildomą įrašą elemento meta duomenyse."
+
+msgid "The purpose for which the rubric is intended to be used. As a reading passage for multiple items, or instructions for a particular section, for example."
+msgstr "Tikslas dėl kurio rubriką turi būti naudojama. Pavyzdžiui, kaip skaitymui skirta ištrauka keliems elementams arba instrukcijos pasirinktai sekcijai."
+
+msgid "The QTI class \"%1$s\" located at line \"%2$d\" is currently not supported."
+msgstr "QTI klasė \"%1$s\" esanti \"%1$s\" eilutėje yra nepalaikoma."
```

Comment:
second argument should be a digit?

Reply:
![image](https://user-images.githubusercontent.com/18699340/66566791-80b84280-eb6e-11e9-9d33-bcc5e563dc4b.png)
yes

Reply:
i will fix it in translate

Reply:
fixed